psp/exp_configs/recent_history.py

```diff
@@ -13,34 +16,164 @@
 PV_DATA_PATH = "data/5min.netcdf"
 NWP_DATA_PATH = "gs://solar-pv-nowcasting-data/NWP/UK_Met_Office/UKV_intermediate_version_7.zarr"
 
-# TODO define
-ExpConfigBase = object
+# A list of SS_ID that don't contain enough data.
+# I just didn't want to calculate them everytime.
+# TODO Get rid of those when we prepare the dataset.
+SKIP_SS_IDS = [
```

Comment:
Have these just been moved from the dataset.py file? And I'm assuming they are only relevant for a specific project?

Reply:
Exactly I moved them to the config.